packages/pn-personafisica-webapp/src/pages/NuovaDelega.page.tsx

```diff
@@ -138,10 +138,13 @@ const NuovaDelega = () => {
 
   const xsValue = isMobile ? 12 : 4;
 
-  useEffect(() => {
+  useEffect(() => () => void dispatch(resetNewDelegation()), []);
+
+  const handleGetAllEntities = () => {
     void dispatch(getAllEntities());
-    return () => void dispatch(resetNewDelegation());
-  }, []);
+  };
+
+  const memoEntities = useMemo(() => entities, [entities]);
```

Comment:
L'utilizzo di memo in questo caso non serve perchè comunque è legato al cambiamento di entities